package.json

```diff
@@ -75,6 +79,7 @@
     "react-with-direction": "^1.1.0"
   },
   "dependencies": {
+    "airbnb-prop-types": "^2.14.0",
```

Comment:
I may have missed it, but is this used anywhere?

Reply:
@ahuth yes! This is where we get `geComponentName` from.

Reply:
https://github.com/airbnb/react-with-styles/pull/221/files#diff-7b388d85c31f5d3c287014af17569273R7